### PR TITLE
Moving rolesanywhere(1), rum(2), s3(1), sagemaker(2), scheduler(2), sqs(2) resources to v1beta1 version

### LINKED ISSUE #535: Moving rolesanywhere(1), rum(2), s3(1), sagemaker(2), scheduler(2), sqs(2) resources to v1beta1 version
* `rolesanywhere`
  * [x] [aws_rolesanywhere_profile](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rolesanywhere_profile)

* `rum`
  * [x] [aws_rum_app_monitor](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rum_app_monitor)
  * [x] [aws_rum_metrics_destination](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rum_metrics_destination)

* `s3`
  * [x] [aws_s3control_storage_lens_configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3control_storage_lens_configuration)

* `sagemaker`
  * [x] [aws_sagemaker_servicecatalog_portfolio_status](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sagemaker_servicecatalog_portfolio_status)
  * [x] [aws_sagemaker_space](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sagemaker_space)

* `scheduler`
  * [x] [aws_scheduler_schedule](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule)
  * [x] [aws_scheduler_schedule_group](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule_group)

* `sqs`
  * [x] [aws_sqs_queue_redrive_allow_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue_redrive_allow_policy)
  * [x] [aws_sqs_queue_redrive_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue_redrive_policy)

### PR DESCRIPTION
**Pull request by steperchuk**
### Description of your changes
* `rolesanywhere`
  * [x] [aws_rolesanywhere_profile](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rolesanywhere_profile)

* `rum`
  * [x] [aws_rum_app_monitor](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rum_app_monitor)
  * [x] [aws_rum_metrics_destination](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rum_metrics_destination)

* `s3`
  * [x] [aws_s3control_storage_lens_configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3control_storage_lens_configuration)

* `sagemaker`
  * [x] [aws_sagemaker_servicecatalog_portfolio_status](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sagemaker_servicecatalog_portfolio_status)
  * [x] [aws_sagemaker_space](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sagemaker_space)

* `scheduler`
  * [x] [aws_scheduler_schedule](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule)
  * [x] [aws_scheduler_schedule_group](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule_group)

* `sqs`
  * [x] [aws_sqs_queue_redrive_allow_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue_redrive_allow_policy)
  * [x] [aws_sqs_queue_redrive_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue_redrive_policy)

Fixes #535

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested
Manually, as resources require real arn to be specified in json blocks
aws_sagemaker_space
aws_sagemaker_space

aws_sqs_queue_redrive_allow_policy
aws_sqs_queue_redrive_allow_policy

aws_sqs_queue_redrive_policy
aws_sqs_queue_redrive_policy

Manually and Uptest
[examples/rolesanywhere/profile.yaml](https://github.com/upbound/provider-aws/actions/runs/4230625993)
[examples/rum/appmonitor.yaml](https://github.com/upbound/provider-aws/actions/runs/4230155188)
[examples/rum/metricsdestination.yaml](https://github.com/upbound/provider-aws/actions/runs/4230155898)
[examples/s3control/storagelensconfiguration.yaml](https://github.com/upbound/provider-aws/actions/runs/4230156446)
[examples/sagemaker/servicecatalogportfoliostatus.yaml](https://github.com/upbound/provider-aws/actions/runs/4230157071)
[examples/scheduler/schedule.yaml](https://github.com/upbound/provider-aws/actions/runs/4230157815)
[examples/scheduler/schedulegroup.yaml](https://github.com/upbound/provider-aws/actions/runs/4230158671)

### CODE REVIEW

**Comment by steperchuk** [comment]
/test-examples="examples/rum/appmonitor.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/rum/metricsdestination.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/s3control/storagelensconfiguration.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/sagemaker/servicecatalogportfoliostatus.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/scheduler/schedule.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/scheduler/schedulegroup.yaml"

File: examples/rum/metricsdestination.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    destination: CloudWatch
+    region: us-east-1
+
+---
+
+apiVersion: rum.aws.upbound.io/v1beta1
+kind: AppMonitor
+metadata:
+  annotations:
+    meta.upbound.io/example-id: rum/v1beta1/appmonitor
```
Comment (dverveiko) [inline examples/rum/metricsdestination.yaml:23]:
```suggestion
    meta.upbound.io/example-id: rum/v1beta1/metricsdestination
```

File: examples/rum/metricsdestination.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    domain: "myDomain.com"
+    region: us-east-1
+
+---
+
+apiVersion: lightsail.aws.upbound.io/v1beta1
+kind: Domain
+metadata:
+  annotations:
+    meta.upbound.io/example-id: rum/v1beta1/appmonitor
```
Comment (dverveiko) [inline examples/rum/metricsdestination.yaml:38]:
```suggestion
    meta.upbound.io/example-id: rum/v1beta1/metricsdestination
```

File: examples/rolesanywhere/profile.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    upjet.upbound.io/manual-intervention: "It requires a real role arn specified."
+    meta.upbound.io/example-id: rolesanywhere/v1beta1/profile
+  labels:
+    testing.upbound.io/example-name: example
+  name: testprofile
+spec:
+  forProvider:
+    name: example
+    region: us-west-1
+    roleArns:
```
Comment (sergenyalcin) [inline examples/rolesanywhere/profile.yaml:14]: What do you think about adding this reference manually and using the selector?

**Review by sergenyalcin — COMMENTED** [review]
Thanks @steperchuk I left a comment

**Comment by steperchuk** [comment]
/test-examples="examples/rolesanywhere/profile.yaml"

**Review by sergenyalcin — APPROVED** [review]
Thanks @steperchuk LGTM!

**Pull request merged**